### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by guhans-argonaut**
Add Argonaut to Tooling - Kubernetes, PAAS, and Cloud Services

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ A curated list of tools and resources for Platform Engineering.
 - [Terraform Guardrails with OPA](https://compellingcloud.substack.com/p/navigating-safety-a-beginners-guide)
 
 ## Tooling— Kubernetes, PAAS and Cloud services
+- [Argonaut - Deploy apps and infrastructure on your cloud in minutes](https://www.argonaut.dev/)
```
Comment (shospodarets) [inline README.md:65]: Can you please remove "in minutes" part to be similar to the https://www.crunchbase.com/organization/argonaut-862d description,
and I will merge

**Comment by shospodarets** [comment]
Thank you for the contribution @guhans-argonaut .
Can you please check the https://github.com/shospodarets/awesome-platform-engineering/pull/2#discussion_r1108086693

File: README.md (hunk 1 above)
Comment (guhans-argonaut) [reply README.md:65]: Sure!

**Comment by guhans-argonaut** [comment]
Changes made @shospodarets.

**Pull request merged**